docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -125,17 +125,19 @@ You can use the following environment variables to specify JVM memory and JVM op
 * `USER_MEM_ARGS` : JVM memory arguments for starting WebLogic Server.
 * `NODEMGR_JAVA_OPTIONS` : Java options for starting a Node Manager instance.
 * `NODEMGR_MEM_ARGS` : JVM memory arguments for starting a Node Manager instance.
+* `WLST_PROPERTIES` : System properties for WLST commands in introspector jobs or WebLogic pods.
+* `WLSDEPLOY_PROPERTIES` : System properties for WebLogic Deploy Tool commands in Model in Image introspector jobs or WebLogic pods.
 
-**Note**: The `USER_MEM_ARGS` environment variable defaults to `-Djava.security.egd=file:/dev/./urandom` in all WebLogic Server pods and the WebLogic introspection job. It can be explicitly set to another value in your domain resource YAML file using the `env` attribute under the `serverPod` configuration.
+**Notes:**
 
-The following behavior occurs depending on whether or not `NODEMGR_JAVA_OPTIONS` and `NODEMGR_MEM_ARGS` are defined:
+* The following behavior occurs depending on whether or not `NODEMGR_JAVA_OPTIONS` and `NODEMGR_MEM_ARGS` are defined:
+  * If `NODEMGR_JAVA_OPTIONS` is not defined and `JAVA_OPTIONS` is defined, then the `JAVA_OPTIONS` value will be applied to the Node Manager instance.
+  * If `NODEMGR_MEM_ARGS` is not defined, then default memory and Java security property values (`-Xms64m -Xmx100m -Djava.security.egd=file:/dev/./urandom`) will be applied to the Node Manager instance. It can be explicitly set to another value in your domain resource YAML file using the `env` attribute under the `serverPod` configuration.
+* The `USER_MEM_ARGS` environment variable defaults to `-Djava.security.egd=file:/dev/./urandom` in all WebLogic Server pods and the WebLogic introspection job. It can be explicitly set to another value in your domain resource YAML file using the `env` attribute under the `serverPod` configuration.
+* Notice that the `NODEMGR_MEM_ARGS` and `USER_MEM_ARGS` environment variables both set `-Djava.security.egd=file:/dev/./urandom` by default. This respectively helps to speed up the Node Manager and WebLogic Server startup on systems with low entropy. 
+* You can use `JAVA_OPTIONS` to disable Fast Application Notifications (FAN), see the [Disable Fast Application Notifications FAQ]({{<relref "/faq/fan.md">}}) for details.
```

Comment:
(FAN), see -> (FAN); see (these are actually two sentences; if you want them joined, then use a semi-colon.)

Reply:
I don't see this one fixed.